wpilib/wpilib/_impl/main.py

```diff
@@ -19,23 +21,30 @@ def _log_versions():
     import wpilib
     import hal
     import hal_impl
-    
+
     import logging
     logger = logging.getLogger('wpilib')
-    
+
     logger.info("WPILib version %s", wpilib.__version__)
     logger.info("HAL base version %s; %s platform version %s",
                 hal.__version__,
                 hal_impl.__halplatform__,
                 hal_impl.__version__)
     if hasattr(hal_impl.version, "__hal_version__"):
         logger.info("HAL library version %s", hal_impl.version.__hal_version__)
-    
+
     # should we just die here?
     if hal.__version__ != wpilib.__version__ and \
        hal.__version__ != hal_impl.__version__:
         logger.warning("Core component versions are not identical! This is not a supported configuration, and you may run into errors!")
-        
+
+    if hal_impl.__hal_simulation__:
```

Comment:
I think `hal.isSimulation()` is the preferred way of spelling this.

Reply:
Ah, didn't notice that. Thanks.